### PR TITLE
Flask web apps - esp

### PR DESCRIPTION
**Pull request by vhvb1989**
Spanish version for Flask web apps (python 101)

### CODE REVIEW

File: es/.vscode/extensions.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (pamelafox) [inline es/.vscode/extensions.json:1]: I didnt realize you could have a .vscode in a subfolder, how does that work? When does VS Code choose to use it?

File: es/.gitignore [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+.sass-cache
+.github
+css
+dist
+examples
+js
+plugin
+gulpfile.js
+package.json
+package-lock.json
```
Comment (pamelafox) [inline es/.gitignore:20]: I'm also fine with all these being added to main .gitignore

**Review by pamelafox — APPROVED** [review]
Amazing! I'll chat with the student ambassadors team about ways to encourage more use of these slides, especially now that they're translated into spanish.

File: es/.vscode/extensions.json (hunk 1 above)
Comment (vhvb1989) [reply es/.vscode/extensions.json:1]: VS Code will only honor and use it when you open the subfolder directly.  The idea is that, if you fork/clone the repo and open it, then you can run `code es` to open the `es` subfolder which loads the Spanish linters and the page you can.

It's a common pattern on Monolithics with many projects in it

File: es/.gitignore (hunk 2 above)
Comment (vhvb1989) [reply es/.gitignore:20]: will move it on next PR

**Pull request merged**